nvm.sh

```diff
@@ -484,11 +484,25 @@ nvm_strip_path() {
     -e "s#${NVM_DIR}/versions/[^/]*/[^/]*${2-}[^:]*##g"
 }
 
-nvm_prepend_path() {
+nvm_change_path() {
+  # if there’s no initial path, just return the supplementary path
   if [ -z "${1-}" ]; then
-    nvm_echo "${2-}"
+    nvm_echo "${3-}${2-}"
+  # if the initial path doesn’t contain an nvm path, prepend the supplementary
+  # path
+  elif [ "$(expr "${1-}" : ".*${NVM_DIR}/[^:]*${2-}.*")" = 0 ] || \
```

Comment:
I've tried to avoid using `expr` in `nvm` because of https://github.com/koalaman/shellcheck/wiki/SC2003. Can this be written differently?